src/Microsoft.ML.Core/Utilities/ResourceManagerUtils.cs

```diff
@@ -29,12 +29,12 @@ public static ResourceManagerUtils Instance
             }
         }
 
-        private const string DefaultUrl = "https://aka.ms/tlc-resources/";
+        private const string DefaultUrl = "https://aka.ms/mlnet-resources/";
```

Comment:
>https://aka.ms/mlnet-resources/ [](start = 43, length = 31)

Did we moved files to this url or not yet?

Reply:
Copying relevant comment of mine from the issue: https://github.com/dotnet/machinelearning/issues/546#issuecomment-423266020
> When doing this, be sure to point the old aka.ms links to the new aka.ms links. The maintains backwards compatibility by ensuring the old links (used in internal builds) still work.
> 
> In addition, pointing the old aka.ms links to the new aka.ms links reduces our long term work, as we then only have to maintain the new links.

Reply:
An update on this: 
- The files remained the same.
- Both the old and the new links point to the same files.

I think we should merge this change now, and then it will take a second to have the old links point to the new links.